### DISCUSSION TITLE
Delete upserted document

### DISCUSSION THREAD

**Discussion by jordicore** [FlowiseAI/Flowise#1088] [Q&A]
Hi everyone,

Can't find out the way of deleting or updating a previously upserted document, any thoughts?

**Comment by HenryHengZJ** [comment]
for now, all document uploaded is stored within the chatflow as base64 encoded string. if you delete the chatflow, it will delete the uploaded docs as well. It wont delete the vector embeddings though

**Comment by jordicore** [comment]
Oh, so currently there is no way to update or delete vector embeddings?
Is there any plans to allow this?
